service/src/main/java/bio/terra/landingzone/library/configuration/LandingZoneSamConfiguration.java

```diff
@@ -11,11 +12,25 @@ public class LandingZoneSamConfiguration {
   /** URL of the SAM instance */
   private String basePath;
 
+  /**
```

Comment:
@rtitle What is the purpose of such users?

Reply:
The issue is Leo creates WSM resources as its own service account (`leonardo-prod@broad-dsde-prod.iam.gserviceaccount.com`), and therefore also needs to query the LZ as its SA. Prior to this change, the Leo SA has no permission to the landing-zone resource, so we were getting errors when creating WSM resources that use a landing zone (e.g. storage container).

We discussed a few options in [the ticket](https://broadworkbench.atlassian.net/browse/TOAZ-290) and settled on this approach, to have a configurable set of system accounts that we always grant user privileges on landing zones.